### PR TITLE
feat<Login,Register,Host,CSS,imports> add the login and  register interface...

### LINKED ISSUE #7: Login functionality
These need to be done:
1. [x] #3 
2. [ ] #10 

Functionality and validations for the login page.

### LINKED ISSUE #10: Login/Sign up Interfaces
Create the Login/Sign up Interfaces

Add the routing to those interfaces in the NavBar

### PR DESCRIPTION
**Pull request by AbiudSM**
..., their respective models and the radzen library

### CODE REVIEW

**Review by netotz — CHANGES_REQUESTED** [review]
Move all the `@code { }` blocks to code behind files (`.razor.cs`)

**Comment by AbiudSM** [comment]
Do you accept my changes? UwU

File: Elecritic/Pages/Login.razor.cs [hunk 1]
```diff
@@ -5,14 +5,28 @@
...
-        [Required]
-        [StringLength(50)]
-        [EmailAddress]
-        public string Email { get; set; }
+    public partial class Login {
 
-        [Required]
-        [StringLength(50)]
-        public string Password { get; set; }
+        LoginRequest model = new LoginRequest();
```
Comment (netotz) [inline Elecritic/Pages/Login.razor.cs]: `private LoginRequest Model { get; set; } = new LoginRequest();`

File: Elecritic/Pages/Register.razor.cs [hunk 2]
```diff
@@ -5,26 +5,39 @@
...
-        [Required]
-        [StringLength(50)]
-        public string Password { get; set; }
-
-        [Required]
-        [StringLength(50)]
-        public string ConfirmPassword { get; set; }
+    public partial class Register {
+
+        RegisterRequest model = new RegisterRequest();
```
Comment (netotz) [inline Elecritic/Pages/Register.razor.cs]: `private RegisterRequest Model { get; set; } = new RegisterRequest();`

**Review by netotz — CHANGES_REQUESTED** [review]

**Review by netotz — DISMISSED** [review]
nice!

**Review by Diegobr21 — DISMISSED** [review]
Nice! We need to review if its necessary for those tasks to run asynchronously as they aren´t currently

**Comment by netotz** [comment]
> We need to review if its necessary for those tasks to run asynchronously as they aren´t currently

@Diegobr21 I think they'll work well when the database calls get implemented, as they will be asynchronous

**Review by netotz — APPROVED** [review]

**Pull request merged**